docs/starters.yml

```diff
@@ -6486,6 +6486,15 @@
     - Desktop and mobile responsive layout
     - Customizable branding & navigation
     - Material-UI
+- url: https://gatsby-starter-moonclerk.netlify.app
+  repo: https://github.com/moonclerk/gatsby-starter-moonclerk
```

Comment:
repos and links are changed?

https://gatsby-starter-payments.netlify.app/
https://github.com/moonclerk/gatsby-starter-payments

Reply:
Yeah I updated them in this PR: 
[https://github.com/gatsbyjs/gatsby/pull/24719](https://github.com/gatsbyjs/gatsby/pull/24719)

Reply:
great :)